City-of-Pori/pori-kada
# PORI-398 Removed footer menu context and settings from visitpori feat…

…ure, added section footer block to front page of visitpori

To test:

1. drush fra -y && drush cc all
2. Go to local.visitpori.fi and make sure there is no menu on footer
3. Go to any sub page of visitpori and make sure there is no footer
4. Go edit the visitpori section footer and make sure it appears on all pages after saving.
